javascript/index.js

```diff
@@ -4,37 +4,166 @@
+            );
+          },
+          (error) => console.log(error)
+        );
+      },
+      (error) => console.log(error)
+    );
+  },
+  (error) => console.log(error)
+);
 
 // Iteration 2 - using promises
-// ...
+obtainInstruction("steak", 0).then((step1) => {
+  document.querySelector("#steak").innerHTML += `<li>${step1}</li>`;
+  obtainInstruction("steak", 1).then((step2) => {
+    document.querySelector("#steak").innerHTML += `<li>${step2}</li>`;
+    obtainInstruction("steak", 2).then((step3) => {
+      document.querySelector("#steak").innerHTML += `<li>${step3}</li>`;
+      obtainInstruction("steak", 3).then((step4) => {
+        document.querySelector("#steak").innerHTML += `<li>${step4}</li>`;
+        obtainInstruction("steak", 4).then((step5) => {
+          document.querySelector("#steak").innerHTML += `<li>${step5}</li>`;
+          obtainInstruction("steak", 5).then((step6) => {
+            document.querySelector("#steak").innerHTML += `<li>${step6}</li>`;
+            obtainInstruction("steak", 6).then((step7) => {
+              document.querySelector("#steak").innerHTML += `<li>${step7}</li>`;
+              obtainInstruction("steak", 7).then((step8) => {
+                document.querySelector(
+                  "#steak"
+                ).innerHTML += `<li>${step8}</li>`;
+                document.querySelector(
+                  "#steak"
+                ).innerHTML += `<li>Stake is ready!</li>`;
+                document.querySelector("#steakImg").removeAttribute("hidden");
+              });
+            });
+          });
+        });
+      });
+    });
+  });
+});
 
 // Iteration 3 using async/await
-// ...
+async function makeBroccoli() {
+  try {
+    let step1 = await obtainInstruction("broccoli", 0);
+    document.querySelector("#broccoli").innerHTML += `<li>${step1}</li>`;
+    let step2 = await obtainInstruction("broccoli", 1);
+    document.querySelector("#broccoli").innerHTML += `<li>${step2}</li>`;
+    let step3 = await obtainInstruction("broccoli", 2);
+    document.querySelector("#broccoli").innerHTML += `<li>${step3}</li>`;
+    let step4 = await obtainInstruction("broccoli", 3);
+    document.querySelector("#broccoli").innerHTML += `<li>${step4}</li>`;
+    let step5 = await obtainInstruction("broccoli", 4);
+    document.querySelector("#broccoli").innerHTML += `<li>${step5}</li>`;
+    let step6 = await obtainInstruction("broccoli", 5);
+    document.querySelector("#broccoli").innerHTML += `<li>${step6}</li>`;
+    let step7 = await obtainInstruction("broccoli", 6);
+    document.querySelector("#broccoli").innerHTML += `<li>${step7}</li>`;
+    let step8 = await obtainInstruction("broccoli", 7);
+    document.querySelector("#broccoli").innerHTML += `<li>${step8}</li>`;
+  } catch (error) {
+    console.log(error);
+    document.querySelector(
+      "#broccoli"
+    ).innerHTML += `<li>Broccoli is ready!</li>`;
+    document.querySelector("#broccoliImg").removeAttribute("hidden");
```

Comment:
This action should be after the "let step8", you declared inside on the catch error function. We should only use that to deal with errors. Overall very good job